Vizzuality/front-end-scaffold
# next@13.2.3

This PR brings the following changes:
- bumps node version: `18.15`
- bumps `next` version: `13.2.3` (and related dependencies: `eslint-config-next`, `@types/react`, `@types/react-dom`)
- updates `browserlist` db.
- removes `next-env.d.ts` from version control. This file is generated when `yarn dev` or `yarn build` are executed.  
![image](https://user-images.githubusercontent.com/999124/223774632-1aa3f017-8ade-4df9-a977-995a85cc5dcf.png)
 (see https://nextjs.org/docs/basic-features/typescript for more info)
- better typing experience in `next.config.js` file: now webpack's configuration is typed.
![image](https://user-images.githubusercontent.com/999124/223775502-22ea20c5-4a22-42a5-9605-53f6004aa6f6.png)


**How to test?**
There is nothing in particular to test: just make a fresh install (remove `node_modules` and `.next` folders, ensure you are using the indicated Node version in `.nvmrc`), install dependencies, run the project. You can also build it, just in case.

Closes #125
